addon/ActivityStreams.js

```diff
@@ -369,11 +369,6 @@ ActivityStreams.prototype = {
    */
   _respondToPlacesRequests({msg, worker}) {
     switch (msg.type) {
-      case am.type("RECENT_LINKS_REQUEST"):
-        PlacesProvider.links.getRecentLinks(msg.data).then(links => {
-          this._processAndSendLinks(links, "RECENT_LINKS_RESPONSE", worker, msg.meta);
```

Comment:
i think recent links is the only one who uses ```_processAndSendLinks``` so you can probably remove that function altogether